src/pkg/immutabletag/dao/model/rule.go

```diff
@@ -1,4 +1,4 @@
-package models
+package model
```

Comment:
Move this into immutabletag/model package if the manager and dao use the same one

Reply:
it's for dao only